File: librarian-app/src/main/java/com/github/dkurata38/liblio/librarian_app/bibliotheca/BibliothecaListController.java
package com.github.dkurata38.liblio.librarian_app.bibliotheca;

import org.springframework.security.core.annotation.AuthenticationPrincipal;
import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;
import com.github.dkurata38.liblio.librarian_app.security.LibrarianUser;

@Controller
public class BibliothecaListController {
    @GetMapping("bibliotheca/bibliotheca-list")
    public String list(@AuthenticationPrincipal LibrarianUser librarianUser) {
        return "bibliotheca/bibliothecaList";
    }
}

File: librarian-app/src/main/kotlin/com/github/dkurata38/liblio/librarian_app/bibliotheca/BibliothecaListController.kt
package com.github.dkurata38.liblio.librarian_app.bibliotheca

import com.github.dkurata38.liblio.librarian_app.security.LibrarianUser
import org.springframework.security.core.annotation.AuthenticationPrincipal
import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping

@Controller
class BibliothecaListController {
    @GetMapping("bibliotheca/bibliotheca-list")
    fun list(@AuthenticationPrincipal librarianUser: LibrarianUser): String {
        return "bibliotheca/bibliothecaList"
    }
}
